metadata/boundaries/county_boundaries

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah County Boundaries
+
+## ID
+
+b2c8af56-3d92-4aa7-ac2f-b1388051d9ce
+
+## Brief Summary
+
+A statewide polygon dataset of county boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+This dataset contains polygons for all 29 counties in Utah, where the perimeter of the polygon represents the county boundary. Each polygon also contains county population as of the [2020 Census](https://www.census.gov/library/stories/state-by-state/utah-population-change-between-census-decade.html) and current approximate population.
+
+## Description
+
+### What is the dataset
+
+This dataset represents each county as a polygon with information such as the Entity Number and FIPS code. A full explanation of the field definitions for this dataset is available on the [schema summary](https://docs.google.com/document/d/1e98nKP6PW9tuTy3nZYS4qhMp9fC8E6uv73SVbBOYvsk/edit?usp=sharing) document.
+
+### What is the purpose of the dataset
+
+This dataset provides an administrative baseline for the current extent of each county in Utah. Boundary data is often used in analysis, especially within the transportation, real estate, and natural resources sectors.
+
+### What does the dataset represent
+
+This dataset does not represent exact legal boundaries, but, rather a set of boundaries used for general purposes that conforms to logical and administrative rules (e.g. no two cities or township areas may cover the same geographic extent).
```

Comment:
"cites or township areas"- should we say "counties" here instead? Do cities and townships affect the county boundaries?

Reply:
I'm not sure whether they do or not, but I think for clarity's sake it would be better for it to simply say "counties". I'll make the change.